Resources/Locale/en-US/ui/actionmenu.ftl

```diff
@@ -0,0 +1,23 @@
+# Action menu stuff (left panel, with hotbars etc)
+
+ui-actionmenu-function-lock-action-slots = (Un)lock dragging and clearing action slots
+ui-actionmenu-function-open-abilities-menu = Open action menu
+ui-actionmenu-function-hotbar1 = Hotbar slot 1
+ui-actionmenu-function-hotbar2 = Hotbar slot 2
```

Comment:
these really should just be `ui-actionmenu-function-hotbar` and then you pass in an argument for the hotbar # (more extensible as well in case more hotbar slots are added) something like

```fluent
ui-actionmenu-function-hotbar = Hotbar slot { $NUMBER }
```

Reply:
actually, are these even used? i don't see a Loc.GetString for them in this PR? i see that these were renamed, but were the loc.getstring calls also renamed to match it?

Reply:
reverted their renaming, it was unnecessary. 
also this is how they are used in OptionsMenu.KeyRebind.cs:
`                    Function = function;
                    var name = new Label
                    {
                        Text = Loc.GetString(
                            $"ui-options-function-{CaseConversion.PascalToKebab(function.FunctionName)}"),
                        HorizontalExpand = true,
                        HorizontalAlignment = HAlignment.Left
                    };`

so I would have to refactor key rebinding if I want to implement adding numbers to hotbar and loadout